### PR TITLE
Make regex string a raw string in python

### PR DESCRIPTION
**Pull request by Emily-Steel**
TL;DR: This prevents `SyntaxError: invalid escape sequence \d`

Longer explanation: Python 3 considers strings to be Unicode rather than ASCII. The `\` in `\d` is interpreted as the beginning of a Unicode character escape sequence rather than left as is as part of the Regex. `\d` is not a valid Unicode sequence so it was left as is so far with a DeprecationWarning thrown. In Python 3.9+ (maybe earlier, haven't tested) it's now a SyntaxError. The easy solution which I have applied here is to use a raw string literal instead of a typical string literal as raw string literals treat backslashes within the string as "just backslashes" and not escaping characters.

### CODE REVIEW

**Comment by cimpeanvasile** [comment]
Is this project still maintained?

**Comment by Emily-Steel** [comment]
Unfortunately it doesn't look like it

**Comment by zbowling** [comment]
@Christopher-Steel Had to make the same patch locally. It looks like Upstream has been abandoned. Bummer. JSONSchema isn't a good alternative.
